docker/docker-compose.yml

```diff
@@ -8,6 +8,17 @@
 version: "3.8"
 
 services:
+  studio:
+    container_name: supabase-studio
+    image: supabase/studio:latest
+    restart: unless-stopped
+    ports:
+      - ${STUDIO_PORT}:3000/tcp
+    environment:
+      - NEXT_PUBLIC_IS_PLATFORM=false
```

Comment:
Even though this is set the login button is taking me to `http://localhost:8080/platform/login` (kong is on port 8000). Need to figure out which env vars are required